### DISCUSSION TITLE
mock the authenticated user?

### DISCUSSION THREAD

**Discussion by bspksal** [prescottprue/cypress-firebase#464] [Q&A] [answered]
The example login shows

```
const uid = "123SomeUid";
cy.login(uid);
```

Is this intended to hit the live firebase server with a uid of an existing user? Or is it a fake uid that returns mock result?

**Comment by prescottprue** [comment, marked as answer]
@bspksal It really hits Firebase's servers for auth (you can have it hit emulators instead), but the string can be any value as Firebase will just add this as a new Firebsae Auth user with the given UID and no auth provider.

This is helpful if you want to associate data in a real database or in the emulators to specific user which you then login as and interact with your application.
